org.lvnauth.LVNAuth.json

```diff
@@ -0,0 +1,110 @@
+{
+    "app-id": "org.lvnauth.LVNAuth",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "startup.sh",
+    "finish-args": [
+	    "--socket=x11",
+	    "--share=network",
+	    "--share=ipc",
+	    "--socket=pulseaudio",
+	    "--filesystem=home",
+	    "--device=dri"
+        ],
+    "modules": [
+        {
+            "name": "lvnauth",
+            "build-commands": [
+		"install -D flatpak/startup.sh ${FLATPAK_DEST}/bin/startup.sh",
+                "chmod +w ${FLATPAK_DEST}/bin/startup.sh",
+                "cp -r src/* ${FLATPAK_DEST}/bin",
+                "install -D flatpak/org.lvnauth.LVNAuth.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -D flatpak/lvnauth_16x16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_64x64.png /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_128x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png",
```

Comment:
Unless you need the smaller sizes, just install 256px and drop the rest

Reply:
it's ok to install them when there are already there.